test/src/js/get-user-status.spec.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
-const { expect } = require('chai');
+
 
 const underTest = require('../../../src/js/get-user-status');
 
 describe('./src/js/get-user-status', function () {
-	it('getUserStatus should be a Function', function () {
-		expect(underTest).to.be.an('function');
+	test('getUserStatus should be a Function', function () {
```

Comment:
Test title "should be function" does not match expectation of an object

Reply:
Fair comment but most of these tests are of no value other than placeholders right now